express-get-json/server.ts

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+
+type Grade = {
+  id: number;
+  name: string;
+  course: string;
+  score: number;
+};
+
+type Grades = {
+  [key: string]: Grade;
+};
+
+const grades: Grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+const app = express();
+
+app.get('/api/grades', (req, res) => {
+  type GradesArray = Grade[];
+  const gradesArray: GradesArray = [];
```

Comment:
The best way to do this is simply:
`const gradesArray: Grade[] = [];`

There's no need to declare an extra type just for this. Also, if you do declare types, it's usually best to put them at the top level of the file.

Reply:
Ill make note for next time